sdks/java/core/src/main/java/org/apache/beam/sdk/io/Read.java

```diff
@@ -119,6 +119,15 @@ private Bounded(@Nullable String name, BoundedSource<T> source) {
       return new Bounded<T>(name, source);
     }
 
+    /**
+     * Returns a new {@link UnboundedReadFromBoundedSource}.
+     *
+     * It performs a unbounded read from the given {@link BoundedSource}.
```

Comment:
What does an "unbounded read" mean in this context? Specify that this means reading a Bounded amount of data into an Unbounded PCollection (not reading from the bounded source repeatedly, for example)